.github/workflows/build.yml

```diff
@@ -479,6 +479,12 @@ jobs:
             image_url: https://github.com/PhotonVision/photon-image-modifier/releases/download/v2025.0.0-beta-6/photonvision_opi5max.img.xz
             cpu: cortex-a8
             image_additional_mb: 1024
+          - os: ubuntu-22.04
+            artifact-name: LinuxArm64
+            image_suffix: rock5c
+            image_url: https://github.com/PhotonVision/photon-image-modifier/releases/download/v2025.0.0-beta-6/photonvision_rock5c.img.xz
```

Comment:
Bump this to beta 7, I'll push that tag tonight

Reply:
Done